orchestration-shared/src/main/java/uk/gov/di/orchestration/shared/entity/ClientRegistry.java

```diff
@@ -232,6 +233,20 @@ public ClientRegistry withSubjectType(String subjectType) {
         return this;
     }
 
+    @DynamoDbAttribute("IsActive")
+    public Boolean getIsActive() {
+        return isActive;
```

Comment:
Will this default to `true` if `isActive` is `null`? Or do we need to use `Optional.ofNullable` like for `PublicKeySource`?

Reply:
Because null should mean `true` here, I think perhaps 
```
if (isActive == null) {
  return true
}
return isActive;
```

 Does the default assignment `private boolean isActive = true;` not cover this though? Or does the `DynamoDbAttribute` decorator overwrite it even if the entry is null?

Reply:
Tested this together in dev and happy the current implementation works as intended